CHANGES/929.feature.rst

```diff
@@ -0,0 +1 @@
+Added `full_name` shortcut for `Chat` object
```

Comment:
I prefer the former because change log is historical data

Present tense looks like TODO list with unfinished tasks:

> Update v1:
> - Add foo
> 
> Update v2:
> - Fix bar

Past tense looks like a list of things already done:

> Update v1:
> - Added foo
> 
> Update v2:
> - Fixed bar